app.js

```diff
@@ -90,6 +91,9 @@ document.querySelector('.addBtn').addEventListener('click', (e) => {
 
 document.querySelector('.listOfBooks').addEventListener('click', (e) => {
   UI.deleteBook(e.target);
-  Storage.removeBook(e.target.parentElement.previousElementSibling.previousElementSibling
-    .textContent.toString().trim());
+  Storage.removeBook(
+    e.target.parentElement.previousElementSibling.textContent
+      .toString()
+      .trim()
+  );
 });
```

Comment:
- [ ] OPTIONAL: This block of code is not correctly updating the localStorage when an item is deleted. Kindly make changes here. :pray: 

![Screenshot from 2021-09-15 21-26-06](https://user-images.githubusercontent.com/64894927/133462779-71754817-737f-437e-9a8b-19b9ef6846bc.png)